tox.ini

```diff
@@ -46,6 +46,40 @@ commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/nvtabular.git
     horovodrun -np 2 sh examples/usecases/multi-gpu/hvd_wrapper.sh python -m pytest -m horovod -rxs tests/unit
 
+[testenv:py38-nvtabular-cpu]
+passenv=GIT_COMMIT
+allowlist_externals = git
+deps =
+    -rrequirements/base.txt
+    -rrequirements/dev.txt
+commands =
+    ; the GIT_COMMIT env is the current commit of the models repo
+    git clone --depth 1 --branch {posargs:main} https://github.com/NVIDIA-Merlin/NVTabular.git nvtabular-{env:GIT_COMMIT}
+    python -m pip install --upgrade "./nvtabular-{env:GIT_COMMIT}"
+    python -m pip install --upgrade -r "./nvtabular-{env:GIT_COMMIT}/requirements/test.txt"
+    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git@{posargs:main}
+    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/dataloader.git@{posargs:main}
+    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/NVTabular.git@{posargs:main}
+    python -m pip install .
+    python -m pytest nvtabular-{env:GIT_COMMIT}/tests/unit
+
+[testenv:py38-systems-cpu]
+passenv=GIT_COMMIT
+allowlist_externals = git
+deps =
+    -rrequirements/base.txt
+    -rrequirements/dev.txt
+commands =
+    ; the GIT_COMMIT env is the current commit of the core repo
+    git clone --depth 1 --branch {posargs:main} https://github.com/NVIDIA-Merlin/systems.git systems-{env:GIT_COMMIT}
```

Comment:
Similar to the other comment - we want to install this commit of `models`, and add a line to install the `main` branch of `systems` below.